pyproject.toml

```diff
@@ -32,7 +32,7 @@ python = ">=3.8,<3.12"
 nautobot = "^2.0.0"
 azure-identity = { version = "^1.15.0", optional = true }
 azure-keyvault-secrets = { version = "^4.8.0", optional = true }
-boto3 = { version = "~1.19.5", optional = true }
+boto3 = { version = "^1.34", optional = true }
```

Comment:
Sorry to keep going around on this, but now I'm wondering if we even need to bump the minimum - should we just make it `^1.19.5` instead?